apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-content.js

```diff
@@ -34,7 +34,7 @@ function getTourContent() {
 		{
 			heading: __( 'Everything is a block', 'full-site-editing' ),
 			description: __(
-				'In the WordPress Editor paragraphs, images, and videos are all blocks.',
+				'In the WordPress Editor, paragraphs, images, and videos are all blocks.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.